pandas/io/formats/format.py

```diff
@@ -873,6 +875,11 @@ def format_array(values, formatter, float_format=None, na_rep='NaN',
         When formatting an Index subclass
         (e.g. IntervalIndex._format_native_types), we don't want the
         leading space since it should be left-aligned.
+    max_colwidth: False, int or None, optional, default None
```

Comment:
can you add a versionadded tag

Reply:
not a public function. have added anyway.